lib/Rx/Observable.php

```diff
@@ -1771,4 +1772,26 @@ public function pluck($property)
             throw new \Exception('Unable to pluck "' . $property . '"');
         });
     }
+
+    /**
+     * Returns an Observable that emits only the first item emitted by the source Observable during
+     * sequential time windows of a specified duration.
+     *
+     * If items are emitted on the source observable prior to the expiration of the time period,
+     * the last item emitted on the source observable will be emitted.
```

Comment:
I was not sure about deviating from their implementation in the first place, but I really needed it to behave this way and can't figure out a use case for the other way. It there a correct way?